crates/sui-graphql-rpc/src/types/address.rs

```diff
@@ -17,6 +17,8 @@ use super::{
     transaction_block::{TransactionBlock, TransactionBlockFilter},
 };
 
+/// The 32-byte address that is either an account address
+/// (derived from a particular signature scheme) or an object ID.
```

Comment:
```suggestion
/// (derived from a particular signature scheme) or an object ID (derived from the digest of the transaction that created the object and from a counter encoding the number of IDs generated by the transaction.)
```

Not sure if we rly need to provide info on how an account or object address is derived